docs/PayPal/integration.md

```diff
@@ -0,0 +1,194 @@
+---
+title: Pay with PayPal Custom Integration
+keywords: 
+contentType: docs
+productStatus: current
+apiVersion: TODO
+sdkVersion: TODO
+---
+
+# Pay with PayPal Custom Integration
+
+Follow these steps to add Card payments.
+
+1. [Know before you code](#know-before-you-code)
+1. [Add PayPal Payments](#add-paypal-payments)
+1. [Test and go live](#test-and-go-live)
```

Comment:
Should these all be 1 or 1, 2, 3?

Reply:
Markdown will auto-number these if all 1 👍